src/shared/components/FluxMonacoEditor.tsx

```diff
@@ -61,7 +80,54 @@ const FluxEditorMonaco: FC<Props> = ({
   wrapLines,
   variables,
 }) => {
-  const connection = useRef<ConnectionManager>(null)
```

Comment:
with this change, make sure your QA includes when there are multiple editors in the same page. Example, in a flow/notebook there can be >1 editor at once.